aviary/validation_cases/benchmark_tests/test_bench_FwFm.py

```diff
@@ -403,6 +403,4 @@ def test_bench_FwFm_SNOPT_MPI(self):
 
 
 if __name__ == '__main__':
-    test = TestBenchFwFmSerial()
```

Comment:
We run this test a lot during debugging, so I prefer having the run commands at the bottom, even if you leave them commented out.